src/__test__/operate.test.js

```diff
@@ -0,0 +1,22 @@
+import operate from '../logic/operate';
+
+describe('Test operate function:', () => {
+  test("operator '+' sums 2 + 3 equals 5", () => {
+    expect(operate('2', '3', '+')).toEqual('5');
+  });
+  test("operator '-' substract 2 - 3 equals -1", () => {
+    expect(operate('2', '3', '-')).toEqual('-1');
+  });
+  test("operator 'x' multiply 2 x 3 equals 6", () => {
+    expect(operate('2', '3', 'x')).toEqual('6');
+  });
+  test("operator '÷' divides 5 ÷ 2 equals 2.5", () => {
+    expect(operate('5', '2', '÷')).toEqual('2.5');
+  });
+  test("operator '%' returns remainder of 5 ÷ 2 equals 1", () => {
+    expect(operate('5', '2', '%')).toEqual('1');
+  });
+  test('Any other operator should throw an error', () => {
+    expect(() => operate('5', '2', '&')).toThrow();
+  });
+});
```

Comment:
Your testing is nearly perfect 😄! As requested in the project requirements, you should test all the calculator functionality. To achieve this result, please make sure to also test:
- [ ] Dividing a number by 0
- [ ] Checking the modulo `%` of equations containing 0

Cheers and keep coding 😉!